pkgs/os-specific/linux/pam_rssh/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  openssl,
+  pam,
+  pkg-config,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "pam_rssh";
+  version = "unstable-2023-01-12";
+
+  src = fetchFromGitHub {
+    owner = "z4yx";
+    repo = pname;
+    rev = "773823b9a8605436e5bcfabd1c3ff8deb8503f43";
+    hash = "sha512-rAr4ugo+IaxOKXtfEq9stK4edodork2uz10trHDv9MN4Oc296uDQjLh8Sle04Z4mbW56rWlT8eHrW6YpqGHFpw==";
+    fetchSubmodules = true;
```

Comment:
Cargo.toml has support for creates from git repos...

Reply:
True, but that requires it to be configured upstream. Currently, they use git submodules directly.